pkgs/top-level/all-packages.nix

```diff
@@ -8423,7 +8425,7 @@ with pkgs;
 
   msitools = callPackage ../development/tools/misc/msitools { };
 
-  multi-ghc-travis = haskell.lib.justStaticExecutables haskellPackages.multi-ghc-travis;
+  haskell-ci = haskell.lib.justStaticExecutables haskellPackages.haskell-ci;
```

Comment:
@basvandijk Why this change?

Reply:
While moving the non-hackage packages to a dedicated file I discovered that `multi-ghc-travis` has been renamed to `haskell-ci` and that `haskell-ci` is now on Hackage. This is explained in: https://github.com/NixOS/nixpkgs/commit/6106664ffab6dd5f6f2c1191969545d6b4f88ee1

Reply:
Mmm maybe an alias might be in order here...